File: Problem Statements/Sum of Digits/SoD.java
import java.util.Scanner;
class SoD{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter a number : ");
        int n=sc.nextInt();
        sc.close();
        int s=0;
        while(n!=0){
            int t=n%10;
            s=s+t;
            n=n/10;
        }
        System.out.print("Sum of Digits : "+s);
    }
}

File: Problem Statements/Sum of Digits/SoD.cpp
#include<iostream>
using namespace std;
void main(){
	int n,s=0,t;
	cout<<"Enter a number : ";
	cin>>n;
	while(n!=0){
		t=n%10;
		s=s+t;
		n=n/10;
	}
	cout<<"Sum of digits : "<<s;
}
